client/src/unifyfs-fixed.c

```diff
@@ -471,6 +482,146 @@ static int unifyfs_split_index(
     return UNIFYFS_SUCCESS;
 }
 
+/*
+ * Clear all entries in the log index.  This only clears the metadata,
+ * not the data itself.
+ */
+static void
+unifyfs_clear_index(void)
+{
+    off_t num_entries = *unifyfs_indices.ptr_num_entries;
+    size_t index_size = sizeof(*unifyfs_indices.index_entry) * num_entries;
+
+    memset(unifyfs_indices.index_entry, 0, index_size);
```

Comment:
For performance, we could skip this memset.  Though it would be nice to have in a debug mode.  Can we conditionally do that?

Reply:
I'll just get rid of it.  There's really no need for the `memset()`